packages/react/src/utils/positioner/Popper.tsx

```diff
@@ -182,13 +212,10 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
   const child =
     typeof children === 'function'
-      ? (children as PopperChildrenFn)({
-          placement: computedPlacement,
-          scheduleUpdate,
-        })
-      : children
+      ? children({ placement: computedPlacement, scheduleUpdate })
+      : (children as React.ReactElement)
 
-  return <Ref innerRef={contentRef}>{React.Children.only(child) as React.ReactElement}</Ref>
+  return <Ref innerRef={contentRef}>{React.Children.only(child)}</Ref>
```

Comment:
Small type improvements there as in real world we allow to pass only `React.ReactElement` (forced via `React.Children.only()`)